update data import instructions

cnolanminich opened the pull request:
updates the SQL to use duckdb, and adds python + R examples

Commit:
update data import instructions

JonMaino17 commented:
Looks good! Do you think we should explicitly point candidates toward the data-import-setup folder in the main README?

cnolanminich commented:
@JonMaino17 yes definitely!  I added you as a collaborator, could you push a commit with the change to this PR?

JonMaino17 committed:
Update README.md

Added link to different data ingestion examples. Also removed mention of Postgres database, since we're encouraging the use of duckdb instead
